src/pytorch_lightning/strategies/ipu.py

```diff
@@ -272,8 +272,13 @@ def to_tensor(x):
     def batch_to_device(self, batch: Any, device: Optional[torch.device] = None, dataloader_idx: int = 0) -> Any:
         # This override is necessary because the cast must occur before the data
         # is moved to the device to prevent wasteful host->device copies.
+        def fp_to_half(tensor):
```

Comment:
Should add a CHANGELOG entry

Reply:
yes.. I was waiting for another Changelog PR to merge..